operations/patch_list.go

```diff
@@ -64,13 +72,32 @@ func PatchList() cli.Command {
 				}
 			}
 
+			if outputJSON {
+				display := make([]restModel.APIPatch, len(patches)-1)
+
+				for _, p := range patches {
+					api := restModel.APIPatch{}
+					api.BuildFromService(p)
```

Comment:
This error needs to be handled (you can probably just return a wrapped copy of it)

Reply:
My bad, didn't realize that even returned an error, I've made this change.

Thanks!